doc/source/cluster/kubernetes/user-guides/tpu.md

```diff
@@ -0,0 +1,71 @@
+(kuberay-tpu)=
+
+# Using TPUs with KubeRay
+
+This document provides tips on TPU usage with KubeRay.
+
+TPUs are available on Google Kubernetes Engine (GKE). To use TPUs with Kubernetes, configure
+both your Kubernetes setup and add additional values to your RayCluster configuration.
+Instructions for configuring a RayCluster with TPUs on GKE can be found here:
+
+- {ref}`kuberay-gke-tpu-cluster-setup`
+
+## Quickstart: Serve a Stable Diffusion model on GKE with TPUs
+
+After setting up a GKE cluster with TPUs and the Ray TPU initialization webhook, you're ready to begin running
+workloads on Ray with TPUs. The :ref:`StableDiffusion example <kuberay-tpu-stable-diffusion-example>` shows how to
+serve a model with Ray on single-host TPUs.
+
+
+## Configuring Ray Pods for TPU usage
+
+Using any TPU accelerator requires specifying `google.com/tpu` resource `limits` and `requests` in the container fields of your `RayCluster`'s
+`workerGroupSpecs`. This resource specifies the number of TPU chips for GKE to allocate each Pod. KubeRay v1.1.0 adds a `numOfHosts`
+field to the RayCluster custom resource, specifying the number of TPU hosts to create per worker group replica. For multi-host worker groups,
+replicas are treated as PodSlices rather than individual workers, with `numOfHosts` worker nodes being created per replica.
+Additionally, GKE uses `gke-tpu` node selectors to schedule TPU pods on the node matching the desired TPU accelerator and topology.
+
+Below is a config snippet for a RayCluster worker group with 2 Ray TPU workers, each scheduled on their own GKE v4 TPU node.
+
+```
+   groupName: tpu-group
+   replicas: 1
+   minReplicas: 0
+   maxReplicas: 1
+   numOfHosts: 2
+   ...
+   template:
+       spec:
+        ...
+        containers:
+         - name: ray-worker
+           image: rayproject/ray:2.9.0-py310
+           ...
+           resources:
+             cpu: "1"
```

Comment:
Can we remove cpu, ephemeral-storage, and memory from this snippet and use `...` instead so that users can focus on the TPU configurations?

Reply:
Done in [e4b38fc](https://github.com/ray-project/ray/pull/45924/commits/e4b38fc0a89cfc9425a707bed0f32f0a53da9171)